pkg/planner/property/physical_property.go

```diff
@@ -297,6 +305,31 @@ func (p *PhysicalProperty) IsSubsetOf(keys []*MPPPartitionColumn) []int {
 	return matches
 }
 
+// NeedEnforceExchangerWithHashByEquivalence checks if the keys can match the needs of partition with equivalence.
```

Comment:
Please add the equivalence comments in here and give a example.

Reply:
It has been updated.